rpm-arm64/Dockerfile

```diff
@@ -0,0 +1,67 @@
+ARG BASE_IMAGE=amazonlinux:2.0.20181114
+FROM ${BASE_IMAGE}
+
+# Build Args
+ARG GIMME_GO_VERSION=1.13.8
+ARG DD_PIP_VERSION=19.1
+ARG DD_SETUPTOOLS_VERSION=41.0.1
+ARG CMAKE_VERSION=3.14.4
+ARG CLANG_VERSION=8.0.0
+ARG DD_TARGET_ARCH=arm64v8
+
+# Environment
+ENV GOPATH /go
+ENV GIMME_GO_VERSION $GIMME_GO_VERSION
+ENV DD_PIP_VERSION $DD_PIP_VERSION
+ENV DD_SETUPTOOLS_VERSION $DD_SETUPTOOLS_VERSION
+ENV CMAKE_VERSION $CMAKE_VERSION
+ENV CLANG_VERSION $CLANG_VERSION
+ENV DD_TARGET_ARCH $DD_TARGET_ARCH
+
+RUN yum groupinstall -y development && yum -y install which perl-ExtUtils-MakeMaker ncurses-compat-libs git procps \
+    curl-devel expat-devel gettext-devel openssl-devel systemd-devel zlib-devel bzip2 glibc-static python-devel tar pkgconfig  \
+    libtool autoconf policycoreutils-python
+
+# RVM
+COPY ./rvm/gpg-keys /gpg-keys
+RUN gpg --import /gpg-keys/*
+RUN rm -rf /gpg-keys
+RUN curl -sSL https://get.rvm.io | bash -s stable
+RUN /bin/bash -l -c "rvm requirements"
+RUN /bin/bash -l -c "rvm install 2.3 && rvm cleanup all"
+RUN /bin/bash -l -c "gem install bundler --no-document"
+
+# Pip & Invoke
+RUN curl "https://bootstrap.pypa.io/get-pip.py" | python2.7 - pip==${DD_PIP_VERSION} setuptools==${DD_SETUPTOOLS_VERSION}
+RUN pip install invoke distro==1.4.0 awscli==1.16.240
```

Comment:
Could we use Python 3 here? Base image doesn't have it so it needs to be installed. Not a blocker but it would useful if we ever need a Python 3 only package.